README.md

```diff
@@ -13,8 +13,17 @@ Getting Started
 
 Prior to running the free5GC operator, Multus needs to be installed on cluster with the macvlan CNI.
 
+You will also need the Nephio API CRDs, from the [api repository](https://github.com/nephio-project/api):
+
+```sh
+git clone https://github.com/nephio-project/api
+kubectl apply -f api/config/crd/bases/ref.nephio.org_configs.yaml
```

Comment:
@tliron we can do this instead

```
kubectl apply -f https://raw.githubusercontent.com/nephio-project/api/main/config/crd/bases/ref.nephio.org_configs.yaml 

kubectl apply -f https://raw.githubusercontent.com/nephio-project/api/main/config/crd/bases/workload.nephio.org_nfdeployments.yaml

kubectl apply -f https://github.com/nephio-project/api/blob/main/config/crd/bases/workload.nephio.org_nfconfigs.yaml
```

Reply:
Ah... this is much better. Actually I should remove the CRD dupes from the free5gc repo! And also fix the Makefile to not install these dupes.